scripts/variantstore/wdl/extract/generate_FOFNs_for_import.py

```diff
@@ -0,0 +1,84 @@
+import os
+import argparse
+
+from terra_notebook_utils import table
+from terra_notebook_utils import workspace
+
+import time
+
+# just keep these around as constants for now
+vcf_files_name = "vcf_files.txt"
+vcf_index_files_name = "vcf_index_files.txt"
+sample_names_file_name = "sample_names.txt"
+error_file_name = "errors.txt"
+
+
+# make a default, but allow a user to overwrite it just in case someone wants to tweak this while tuning
+attempts_between_pauses=500
+
+def generate_FOFNs_from_data_table(data_table_name, sample_id_column_name, vcf_files_column_name, vcf_index_files_column_name):
+
+    vcf_files = open(vcf_files_name, "w")
+    vcf_index_files = open(vcf_index_files_name, "w")
+    sample_names_file = open(sample_names_file_name, "w")
+    error_file = open(error_file_name, "w")
```

Comment:
this is a job for multi-`with`, [something like](https://stackoverflow.com/questions/893333/multiple-variables-in-a-with-statement):

```
with open(vcf_files_name, "w") as vcf_files, open(vcf_index_files_name, "w") as vcf_index_files ...
```

and then the `closes` on lines 54 - 57 can be deleted. The `with` will take care of closing files automatically, even if the enclosed block exits abnormally.